specification/walkthrough.md

```diff
@@ -0,0 +1,263 @@
+
+However, since we want to model each `BOLD` image separately, we must `GroupBy` **both *subject* and *run***, resulting in six groups with a single image each. 
+
+### Subject level Node
+
+#### From Run Outputs to Subject Inputs
+
+Next, we define a fixed-effects model to combine contrast outputs from each subject's runs together. 
+
+```{note}
+By default, `Nodes` are linked sequentially, with all the `Contrast` outputs from a `Node` available to the subsequent `Node`. 
+```
+
+We need to use `GroupBy` to define how to group the outputs from the `Run` node as inputs to the `Subject` level:
+
+```json
+      "Level": "Subject",
+      "Name": "subject_level",
+      "GroupBy": ["subject", "contrast"],
+```
+
+Here we are specifying that all images belonging to a single `subject` and from a single `contrast` should be grouped together for analysis.
+
+Note that with 3 subjects and 2 runs, we will have 6 groups of output images from the `Run` node. Given two types of images (`variance` and `effect`), this results in 12 images that would be grouped as follows:
+
+
+| image       | subject     | run | contrast |
+| ----------- | ----------- | ----------- | ----------- |
+| sub-01_task-simon_run-1_contrast-IvC-effect_statmap.nii.gz | 01 | 1 | IvC |
+| sub-01_task-simon_run-1_contrast-IvC-variance_statmap.nii.gz | 01| 1| IvC |
+| sub-01_task-simon_run-2_contrast-IvC-effect_statmap.nii.gz | 01| 2| IvC |
+| sub-01_task-simon_run-2_contrast-IvC-variance_statmap.nii.gz |01| 2| IvC |
+| - | - | - | - |
+| sub-02_task-simon_run-1_contrast-IvC-effect_statmap.nii.gz | 02| 1| IvC |
+| sub-02_task-simon_run-1_contrast-IvC-variance_statmap.nii.gz | 02| 1| IvC |
+| sub-02_task-simon_run-2_contrast-IvC-effect_statmap.nii.gz | 02| 2| IvC |
+| sub-02_task-simon_run-2_contrast-IvC-variance_statmap.nii.gz | 02| 2| IvC |
+| - | - | - | - |
+| sub-03_task-simon_run-1_contrast-IvC-effect_statmap.nii.gz | 03| 1| IvC |
+| sub-03_task-simon_run-1_contrast-IvC-variance_statmap.nii.gz | 03| 1| IvC |
+| sub-03_task-simon_run-2_contrast-IvC-effect_statmap.nii.gz | 03| 2| IvC |
+| sub-03_task-simon_run-2_contrast-IvC-variance_statmap.nii.gz | 03| 2| IvC |
+
+```{tip}
+In this example there is only one `contrast`, but we include `contrast` as a grouping variable to be explicit.
+```
+
+#### Subject-level Model
+
+We can now specify the `Subject` level `Model`. Since our intent is to estimate the *mean* for each subject, we only need an intercept (`1`) in our model. We specify the `"Type"` to be `Meta`, which is a special type to identify fixed-effects models.
+
+Remember that we must specify `Contrasts` in order to produce outputs for the next `Node`. Here, we compute a simple identity identity contrast to pass forward the subject-evel estimates forward. Note that we specified the `Test` as `pass`, since we don't want to perform a t-test, but simply need pass forward parameter and variance estimates.   
```

Comment:
```suggestion
Remember that we must specify `Contrasts` in order to produce outputs for the next `Node`. Here, we compute a simple identity contrast to pass forward the subject-evel estimates forward. Note that we specified the `Test` as `pass`, since we don't want to perform a t-test, but simply need to compute and pass forward parameter and variance estimates.
```